emission/tests/funcTests/TestFunctionTiming.py

```diff
@@ -67,20 +68,21 @@ def execute_and_time_function(func: t.Callable[[], bool]):
         logging.info(f"Function '{function_name}' executed successfully in {elapsed_ms:.2f} ms.")
 
         # Verification: Adjusted Query to Match Document Structure
-        timeseries_db = builtin_ts.get_timeseries_db(key="stats/dashboard_time")
-        
+        timeseries_db = esta.TimeSeries.get_time_series(None)
 
-        query = {
-            "metadata.key": "stats/dashboard_time",
-            "data.name": function_name,
+        # Define the time range and additional filters
+        time_query = {
```

Comment:
we should not be hardcoding the `time_query` in mongodb format. Please look at prior examples of how queries are implemented.